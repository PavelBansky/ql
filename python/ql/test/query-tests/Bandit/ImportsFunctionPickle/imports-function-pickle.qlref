Bandit/imports-function-pickle.ql