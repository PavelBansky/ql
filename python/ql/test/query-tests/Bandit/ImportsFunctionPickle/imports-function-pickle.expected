| Test.py:4:10:4:29 | __import__() | Consider possible security implications associated with pickle module |
| Test.py:17:5:17:37 | Attribute() | Consider possible security implications associated with pickle module |
